scripts/split_repo_migration/single-library.git-migrate-history.sh

```diff
@@ -179,3 +179,4 @@ else
 fi
 
 popd
+echo "${DISTRIBUTION_NAME}"
```

Comment:
Is this for debugging purposes? If not, I'd add more context to print statement. If it's, I don't think it should be committed.

Reply:
I added a comment. Also see the internal batch migration script.